### ISSUE TITLE
Unable to change pano-id in 

### ISSUE THREAD

**Issue by gvdkooij** [pchen66/panolens.js#314]
##### Description

I want to embed a Google Street View image with Panolens. The original file is working fine, but when I want to use another panorama it is not working. 

This one is working:
https://codepen.io/gvdkooij/pen/xxRQJMY

When i change the Google panorma ID to "FEpIJhSgOzoAAAQJOQCL3w" with this method https://stackoverflow.com/questions/29916149/google-maps-streetview-how-to-get-panorama-id
I see only a black screen: 
see: https://codepen.io/gvdkooij/pen/eYBQjaR

I tried to add my Google Maps JavaScript API to  the code.
see: https://codepen.io/gvdkooij/pen/abBQazG

But the only thing I see is a black screen.
What am I doing wrong.

##### Panolens version

- [X] Master
- [ ] Dev

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

**Comment by flyandi** [comment]
I don't think this is a library issue. I only spent 10min time on this but the ID is not correct. There is a whole lot confusion around what the ID is and what is not and Google seem to change them at their discretion. So I think you need to research a bit what ID they allow - also I saw that some pictures are not public, so Google will throw a 403 if directly accessed.

Btw all the IDs above throw a 404 not found from Googles side.

**Comment by gvdkooij** [comment]
I think you are right, I did a lot of research about the Pano ID and tried many methods to get a good PanoID, with a Pano from 2016 or newer. But there seems to me no way to do this. Do you have a panoID in the new system, which is working?

**Comment by gvdkooij** [comment]
I hope there will be a solution for this problem.....
The library is only working with old PANO-ID's and there is no solution for new PANO-id's.
